scripts/socks/socks.sh

```diff
@@ -18,8 +18,8 @@ AUTOCONFIG_PORT=9500
 # from https://stackoverflow.com/a/246128/4907881
 PROXY_PAC_LOCATION="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Also out of scope, but instead of having to host this locally, we could just directly reference the proxy.pac file on GitHub: https://raw.githubusercontent.com/department-of-veterans-affairs/va.gov-team/master/scripts/socks/proxy.pac.